tests/test_e2e_50_maintenance.py

```diff
@@ -1222,3 +1224,128 @@ def test_120_extend_ended_mw_on_switch_should_fail(self):
         api_url = KYTOS_API + '/maintenance/v1/' + mw_id + '/extend'
         response = requests.patch(api_url, data=json.dumps(payload2), headers={'Content-type': 'application/json'})
         assert response.status_code == 400, response.text
+
+    def test_125_multiple_payload_item_filtering(self):
+        self.net.start_controller(clean_config=True, enable_all=True)
+        self.net.wait_switches_connect()
+        time.sleep(10)
+
+        start = datetime.now(pytz.utc) + timedelta(days=1)
```

Comment:
@HeriLFIU, `pytz` is indeed an existing indirect `maintenance` dependency. But, on e2e let's stick with only `datetime.utcnow()` or `datetime.now(timezone.utc)`, as you've done on other PRs https://github.com/kytos-ng/kytos-end-to-end-tests/blob/master/tests/test_e2e_50_maintenance.py#L146

In the future we have plans to only support UTC, so let's let's try to avoid having `pytz` here too, although it's only being used for utc, but let's keep with only the datetime utc from stdlib, OK?